.github/workflows/build-containers.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           cache-to:   type=registry,ref=ghcr.io/${{ github.repository_owner }}/star-sw-${{ matrix.starenv }}:cache,mode=max
           build-args: starenv=${{ matrix.starenv }}
           push: true
-          tags: ghcr.io/${{ github.repository_owner }}/star-sw-${{ matrix.starenv }}:${{ steps.branch-name.outputs.current_branch }}_latest
+          tags: ghcr.io/${{ github.repository_owner }}/star-sw-${{ matrix.starenv }}:${{ steps.branch-name.outputs.current_branch }}
```

Comment:
I can't remember why the `_latest` suffix came around, but I'd guess it is there for a reason. Could be there because we used to create tag matching the branch name `SLXXy` instead of a different name `SLXXy_0`. Also it's nice to be able to easily distinguish tags that are "permanent" references and the branches for which things might be in flux.

Reply:
You remember it right. The `_latest` suffix indeed helped to distinguish between conflicting branch and tag names. But we are not using the same name for branches and tags anymore because it is very confusing. The tags now have an underscore in their name just like you mentioned. It is better to fix this notation now rather than continue to support the bad choices from the CVS times.

Reply:
As you may have noticed, my intention is to move the label from the package name to the image tag. In the future we may introduce a more conventional `:latest` tag